packages/shell-api/src/field-level-encryption.ts

```diff
@@ -350,47 +350,36 @@ export class KeyVault extends ShellApiWithMongoClass {
     return await makeSingleDocReturnValue(() => this._keyColl.find({ 'keyAltNames': keyAltName }), 'KeyVault.getKeyByAltName', this._instanceState);
   }
 
+  // eslint-disable-next-line @typescript-eslint/require-await
   @returnType('Cursor')
   @apiVersions([1])
   @returnsPromise
   async getKeys(): Promise<Cursor> {
-    return this._keyColl.find({});
+    return new Cursor(
```

Comment:
`libmongocrypt.getKeys` returns a `FindCursor` which is why I am wrapping that up in a `Cursor` here. I had doubts about `returning FindCursor instead of Cursor` being an API change. Let me know.

Reply:
Yes, this should be using `new Cursor` to return a shell cursor :+1: